README

```diff
@@ -66,6 +66,36 @@ In addition to the usual options, configure understands the following:
        this value is actually supplied as "\\\\".
 
 
+Building on macOS
+-----------------
+
+The following instructions apply to both Apple Silicon and Intel Macs.
+
+To build plptools on macOS, you need to install the GNU autotools and the
```

Comment:
You shouldn't need to install autotools unless installing from Git (and then you need autotools on every platform), so I think you can delete this bit.

Reply:
I agree, the section above already states that GNU autotools are necessary to build from Git.